rkyv/src/impls/bitvec.rs

```diff
@@ -173,18 +145,18 @@ mod tests {
     };
     use bitvec::prelude::*;
 
-    #[cfg(feature = "bitvec_std")]
     #[test]
+    #[cfg(feature = "bitvec_alloc")]
     fn bitvec() {
-        use crate::ser::serializers::AllocSerializer;
+        use crate::ser::serializers::CoreSerializer;
 
-        let mut serializer = AllocSerializer::<256>::default();
+        let mut serializer = CoreSerializer::<256, 256>::default();
```

Comment:
It seems that using `CoreSerializer` causes the test to fail, the archived version just becomes an empty slice. I replaced the `AllocSerializer` by a `CoreSerializer` in `tiny_vec::tests::tiny_vec` and got a similar error.

Did you change the `AllocSerializer` to a `CoreSerializer` by mistake or should it be possible to archive heap-allocated types with it?

Reply:
Ah, good spot. `CoreSerializer` doesn't work right with `archived_root` out of the box. I'll fix that up, do a once-over, and then this should be good to merge.